frontend/website/src/Developers.re

```diff
@@ -0,0 +1,179 @@
+module Styles = {
+  open Css;
+  let page =
+    style([
+      marginLeft(`auto),
+      marginRight(`auto),
+      media("(min-width: 60rem)", [maxWidth(`rem(50.66))]),
```

Comment:
can we stick this media query string somewhere where we can share it across all the files on this page (and if it's standard for future pages, future pages)

Suggestion: `Style.re` under the `MediaQuery` module (though you can maybe even make another submodule there called `Standard` or something since we're running out of names) and then you can name these `tablet` and `desktop`?